srcpkgs/googlemaps/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'googlemaps'
+pkgname=googlemaps
+version=.0.0.0.2
```

Comment:
That initial dot doesn't belong in the version number.

Reply:
Done